### PR TITLE
Manual sync from AK/3.0

### PR DESCRIPTION
**Pull request by ableegoldman**
Need to sync over this small patch to fix the build in preparation for a customer hotfix

Merge contains this commit: https://github.com/apache/kafka/commit/8425daf92db54e87727600c0a42323f46e52cfc0

### CODE REVIEW

**Review by gracechensd — APPROVED** [review]

**Review by gracechensd — COMMENTED** [review]
Actually there should not be a 7.0.x in this repo, `7.0.x` is not the "hotfix branch" (release eng will create the actual hotfix branch in our own time). I believe we can just close this? / should this be in ce-kafka instead?

**Comment by ableegoldman** [comment]
Ah ok in that case I'll need one of the code owners to delete this branch, I guess it's been a long while since I was involved in a hotfix and things have changed...sorry about that

As for the ce-kafka matter, I guess I still have concerns about giving the customer a ce-kafka bundle when they were running on ccs-kafka before: are we allowed to do that, would there be trouble with licensing, etc. But, clearly my understanding of the hotfix procedure is wildly out of date, so I'll work on getting this synced to ce-kafka and defer to whatever you think is right. 

Thanks

**Comment by gracechensd** [comment]
I'll handle the 7.0.x deletion. We do probably need this in ccs kafka, just not in a 7.0.x branch if that makes sense - need this commit on a temporary branch somewhere and then release eng will cherry-pick that into the hotfix build when we're ready

**Pull request closed without merging**
